apps/vanilla/src/pages/index.html

```diff
@@ -4,13 +4,113 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./index.css" />
-    <title>Vite App</title>
+    <title>Books App</title>
   </head>
-  <body class="default-layout">
-    <h1 class="title">Main page ⚡</h1>
-    <div class="links">
-      <a href="example/">Go to example page</a>
-    </div>
+  <body>
+
+    <header>
+			<p class="logo">📚 Books app</p>
+		</header>
+
+		<main class="container">
+
+			<div class="filters-container">
+				<p class="subtitle">Filters</p>
+				<form class="filters-form">
+					<input type="text"
+								 name="search-field"
+								 placeholder='e.g. "Refactoring"'
+								 class="form-field search-field">
+					<select name="sort-field" class="form-field sort-field-select">
+						<option value="">Choose sort field</option>
+						<option value="Sort field">Sort field</option>
+						<option value="Another sort field">Another sort field</option>
+					</select>
+					<label class="checkbox-container">Show comics
+						<input type="checkbox" checked="checked">
+						<span class="checkmark"></span>
+					</label>
+				</form>
+			</div>
+
+			<div class="content">
+
+				<section class="content__year">
+					<p class="subtitle year-title">2020 Year</p>
+					<div class="books-container">
+						<div class="book-container">
+							<img src="./assets/images/image 1.png"
+									alt="Book cover"
+									class="book-cover">
+							<p class="book-title">Refactoring UI</p>
+						</div>
+						<div class="book-container">
+							<img src="./assets/images/image 2.png"
+									alt="Book cover"
+									class="book-cover">
+							<p class="book-title">Master the Skill of Debugging CSS</p>
+						</div>
+						<div class="book-container">
+							<img src="./assets/images/image 3.png"
+									alt="Book cover"
+									class="book-cover">
+							<p class="book-title">Blue Period</p>
+						</div>
+					</div>
+				</section>
+
+				<section class="content__year">
+					<p class="subtitle year-title">1980 Year</p>
+					<div class="books-container">
+						<div class="book-container">
+							<img src="./assets/images/image 4.png"
+									 alt="Book cover"
+									 class="book-cover">
```

Comment:
```suggestion
									alt="Book cover"
									class="book-cover">
```

Incorrect indents
Please, correct them in the entire file